zebedee-cms/src/main/java/com/github/onsdigital/zebedee/api/CollectionDetails.java

```diff
@@ -83,6 +83,8 @@ public CollectionDetail get(HttpServletRequest request, HttpServletResponse resp
             collection.description.teams.add(team.getName());
         });
 
+        result.instances = collection.description.getInstances();
```

Comment:
If there is a getDescription method can you use that instead please?

Reply:
👍